File: integration_tests_old/csharp/user/Location.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;
using Newtonsoft.Json.Linq;

namespace Jtd.JtdCodegenDemo
{
    /// <summary>
    /// A latitude / longitude pair indicating a position on Earth
    /// </summary>
    
    public class Location 
    {


        /// <value>
        /// Latitude
        /// </value>
        [JsonProperty("lat")]
        public string Lat { get; set; }

        /// <value>
        /// Longitude
        /// </value>
        [JsonProperty("lng")]
        public string Lng { get; set; }





    }
}

File: integration_tests_old/java/user/Location.java
package com.jsontypedef.jtdcodegendemo;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonSubTypes;
import com.fasterxml.jackson.annotation.JsonTypeInfo;
import com.fasterxml.jackson.annotation.JsonValue;

import java.time.OffsetDateTime;
import java.util.List;
import java.util.Map;

/**
 * A latitude / longitude pair indicating a position on Earth
 */

public class Location {

  /**
   * Latitude
   */
  
  @JsonProperty("lat")
  private String lat;

  /**
   * Longitude
   */
  
  @JsonProperty("lng")
  private String lng;


  
  public Location() {
  }
  


  /**
   * Latitude
   */
  public String getLat() {
    return lat;
  }

  /**
   * Latitude
   */
  public void setLat(String lat) {
    this.lat = lat;
  }

  /**
   * Longitude
   */
  public String getLng() {
    return lng;
  }

  /**
   * Longitude
   */
  public void setLng(String lng) {
    this.lng = lng;
  }

}
